example-app/src/main/java/com/adyen/checkout/example/data/storage/SharedPreferencesEntry.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 29/8/2024.
+ */
+
+package com.adyen.checkout.example.data.storage
+
+enum class SharedPreferencesEntry(val key: String, val defaultValue: Any?) {
```

Comment:
My intention with separating this class from settings is that it can contain non settings related prefs as well. If you check its usages you can see that the settings related classes do not directly access it. WDYT?

Reply:
I was thinking of keeping preferences separate for each screen/flow. But the list is not that long, so I think it is fine to keep it in one place.